master/buildbot/scripts/runner.py

```diff
@@ -487,13 +488,14 @@ class CheckConfigOptions(base.SubcommandOptions):
 
     def getSynopsis(self):
         return "Usage:\t\tbuildbot checkconfig [configFile]\n" + \
-            "\t\tIf not specified, 'master.cfg' will be used as 'configFile'"
+            "\t\tIf not specified, the config file specified in " + \
+            "'buildbot.tac' from the current directory will be used"
 
     def parseArgs(self, *args):
         if len(args) >= 1:
             self['configFile'] = args[0]
         else:
-            self['configFile'] = 'master.cfg'
+            self['configFile'] = os.getcwd()
```

Comment:
yes checkconfig() expects configFile to contain a path to the config file or a directory.
How about setting it to None and then using os.getcwd() if it is not set in checkconfig()?